app/uk/gov/hmrc/emcstfe/models/request/SubmitCreateMovementRequest.scala

```diff
@@ -74,4 +79,22 @@ case class SubmitCreateMovementRequest(body: CreateMovementModel)
       case _ =>
         body.consignorTrader.countryCode
     }
+
+  override def eisXMLBody(): String = {
+    withEisMessage(
+      body = body,
+      messageNumber = messageNumber,
+      messageSender = messageSender,
+      messageRecipient = messageRecipient,
+      messageIdentifier = draftId
```

Comment:
Even though we won't be using ChRIS for the submission, I wonder if it's still worth populating the `draftId` into  https://github.com/hmrc/emcs-tfe/blob/ETFE-2533/app/uk/gov/hmrc/emcstfe/models/request/SoapEnvelope.scala#L64-L66 for completeness?

Reply:
I could do that, would we set the message identifier for ChRIS only in the CAM flow? And the rest will be messageUUID?

Reply:
Yes I believe that's the best approach and mirror's legacy.

Reply:
Done in [6f2b939](https://github.com/hmrc/emcs-tfe/pull/96/commits/6f2b939ed4983fc8b3c058de67a887a5dcb12f16)